src/main/kotlin/uk/gov/gdx/datashare/config/S3ClientsConfig.kt

```diff
@@ -5,7 +5,6 @@ import com.amazonaws.auth.BasicAWSCredentials
 import com.amazonaws.client.builder.AwsClientBuilder
```

Comment:
I think we might be able to just delete tis whole thing, it looks like the AmazonS3 bean is only used in files that are deleted